embark.json

```diff
@@ -11,8 +11,7 @@
   },
   "versions": {
     "web3": "1.2.4",
```

Comment:
You can remove web3 too. We no longer expose that change of version either iirc. (cc @michaelsbradleyjr )